src/plugin/field_base.cpp

```diff
@@ -136,6 +147,48 @@ namespace chapel {
     }
   }
 
+  string FieldGeneratorBase::default_value(const FieldDescriptor* descriptor) {	
+    switch (descriptor->type()) {
+      case FieldDescriptor::TYPE_MESSAGE:
+        return ""; // we use chapel compiler generated default values for messages;
+      case FieldDescriptor::TYPE_ENUM:
+        return "0:" + GetEnumName(descriptor->enum_type());
+      case FieldDescriptor::TYPE_INT64:
+        return StrCat(descriptor->default_value_int64());
```

Comment:
We just changed the other code to avoid using StrCat. Probably makes sense to either make these use `std::to_string` or un-do the other change.

Reply:
Yes, I have that in mind. I will make that change directly to the master branch :)